chsdi/tests/integration/test_layers.py

```diff
@@ -0,0 +1,108 @@
+# -*- coding: utf-8 -*-
+
+import os
+from webtest import TestApp
+from pyramid.paster import get_app
+from sqlalchemy import distinct
+from sqlalchemy.orm import scoped_session, sessionmaker
+from sqlalchemy.sql.expression import func
+
+from chsdi.models.bod import LayersConfig
+from chsdi.models import models_from_name
+
+
+class LayersChecker(object):
+
+    def __enter__(self):
+        def num(s):
+            try:
+                return long(s)
+            except ValueError:
+                return s
+
+        self.testapp = TestApp(get_app('development.ini'))
+        self.session = scoped_session(sessionmaker())
+
+        # configuration via environment. Default is None for all
+        self.staging = os.environ.get('CHSDI_STAGING') if os.environ.get('CHSDI_STAGING') is not None else 'prod'
+        self.onlylayer = os.environ.get('CHSDI_ONLY_LAYER')
```

Comment:
Minor:
I would transform that to a list per default and split on  a separator so that we can test several layers using one variable.

Just change the test to q[0] not in self.onlylayers etc..